### ISSUE TITLE
NFR: Search Connector Upgrade for Solaris

### ISSUE THREAD

**Issue by mollyaljenkins** [hlxsites/vg-macktrucks-com#617]
**Description** 
The AEM team on Volvo needs to perform an upgrade of the Search Connector that affects the Solaris branch used for Volvo EDS. AEM team is blocked by this task. The upgrade does not foresee a direct impact on search functionality.  

Deadline for upgrade: (pending feedback from AEM team) 

**Requirements**  

- Availability of Girish for 1-2 days of regression testing 
- To be planned into 24A - Sprint 4

**Comment by BeckyMedlin** [comment]
Per discussion with Frank Fernow, this is no longer needed. @mollyaljenkins

**Comment by mollyaljenkins** [comment]
Duplicate of https://github.com/hlxsites/vg-volvotrucks-us/issues/583
